client/src/pages/Home.js

```diff
@@ -124,9 +124,15 @@ const HomeTiles = props => {
   return (
     <Grid fluid>
       <Row>
-        {queries.map((query, index) => (
-          <HomeTile key={index} query={query} setSearchQuery={setSearchQuery} />
-        ))}
+        {queries
+          .filter(q => q.query != null)
```

Comment:
Ha, our eslint rule does not pick this one up. Ill tighten the rule (and fix this one and others).